src/js/classes/Graph.js

```diff
@@ -486,7 +490,7 @@ export class DistributionGraph {
     this.distributionIdx = 5;
     // Sort data by county name
     this.data.sort((a, b) =>
-      a["data"][this.nameIdx] > b["data"][1]
+      a["data"][this.nameIdx] > b["data"][this.nameIdx]
```

Comment:
Has this always been wrong lol?

Reply:
Not wrong, just not using the idx variable